lemur/tests/test_certificates.py

```diff
@@ -153,8 +153,12 @@ def test_cleanup_after_revoke(session, issuer_plugin, crypto_authority):
     assert not revoked_cert.destinations
 
 
-def test_get_by_attributes(session, certificate):
-    from lemur.certificates.service import get_by_attributes
+def test_get_by_attributes(session, authority, user, certificate):
```

Comment:
Most of the changes in this file relate to making it able to run independently of all other tests. Without these changes, the test was failing because it depended on state established by `test_authorities.py`; it is now independent, and passes on its own.